public/js/filterbar.js

```diff
@@ -34,8 +35,10 @@ $(function() {
 				}
 				return true;
 			}
-		}).show();
-
+		});
+		filteredRows.show();
+		var emptySearch = $('.empty-search');
+		filteredRows.length === 0 ? emptySearch.removeAttr('hidden') : emptySearch.attr('hidden');
```

Comment:
If you switch to using the `aria-hidden` attribute, you'll just need to update that attribute to `true/false` instead.